transport/internet/sockopt_linux.go

```diff
@@ -14,7 +14,7 @@ const (
 )
 
 func bindAddr(fd uintptr, address net.Address, port net.Port) error {
-	err = syscall.SetsockoptInt(fd, syscall.SOL_SOCKET, syscall.SO_REUSEADDR, 1)
+	err := syscall.SetsockoptInt(int(fd), syscall.SOL_SOCKET, syscall.SO_REUSEADDR, 1)
```

Comment:
not sure if casting from `uintptr` to `int` is the right thing to do though. does not feel right to cast a pointer to int.